webview/src/setup/components/remotes/AzureBlobStorage.tsx

```diff
@@ -15,7 +15,7 @@ export const AzureBlobStorage = () => (
       <a href="https://learn.microsoft.com/en-us/azure/storage/blobs/versioning-overview">
         Blob versioning
       </a>{' '}
-      enabled on the storage account and container.
+      enabled on the storage account{'\n'}and container.{' '}
```

Comment:
Maybe it has been already discussed in a previous pr, but why are we adding these new lines in the first place? I'm not seeing the docs use new lines unless they're starting a new paragraph entirely. 

<img width="727" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/a4ba2dc4-4fca-492b-8e53-2426e6b7e1d1">

Are we doing it to help emphasize certain lines? If so, maybe we should consider rewording text or using methods such as making keywords **bold**/*italic*?